data/remnant/remnant.txt

```diff
@@ -475,6 +480,34 @@ phrase "friendly remnant"
 		"."
 		"!"
 
+phrase "friendly remnant"
+	word
+		"Please"
+	word 
+		" "
+	word
+		"do not"
+		"try not to"
+		"refrain from"
+	word
+		" "
+	word
+		"scanning"
```

Comment:
This phrase doesn't make sense 2 thirds of the time, "do not scanning us" and "try not to scanning" don't work. Perhaps replace `"refrain from"` with some thing else or delete it and change `"scanning"` to `"scan"` or reword the other 2 to make sense with `"scanning"`?

Reply:
Hmm... Must have been half asleep when I wrote that. Thanks for pointing it out!